.vscode/settings.json

```
@@ -1,5 +1,8 @@
 {
     "cSpell.words": [
         "unprovide"
-    ]
+    ],
+    "eslint.workingDirectories": [
+    "./src/Umbraco.Web.UI.Client/"
```

Maybe add Login as well?

```suggestion
    "./src/Umbraco.Web.UI.Client/",
    "./src/Umbraco.Web.UI.Login/"
```